sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GoneAndRetryWithRetryPolicy.java

```diff
@@ -5,12 +5,12 @@
 
```

Comment:
Please consider sorting the imports here as one of them may now be out of order.

Reply:
Done